deployments/apiv2/services/users/src/models/users.py

```diff
@@ -85,14 +92,19 @@ class UserProfile(BaseModel):
     email: EmailStr
     user_id: str
     account_type: str
-    scope: list
+    scope: List[str]
 
 
 class CustomerProfile(BaseModel):
     email: EmailStr
     user_id: str
-    scope: list
+    scope: List[str]
 
 
 class UserAction(BaseModel):
     action_type: str
+
+
+class LoginResponse(BaseModel):
```

Comment:
I think we'll need to update the desktop app with this change, and potentially some of the other changes as well

Reply:
before merging this to main right?

Reply:
Yeah we should just make a ticket to do it